### PR TITLE
chore: codemods for /types entry

### PR DESCRIPTION
**Pull request by talkor**
https://monday.monday.com/boards/3532714909/pulses/7303260067

### CODE REVIEW

File: packages/codemod/types/paths.types.ts [hunk 1]
```diff
@@ -1,3 +1,3 @@
-import { CORE_IMPORT_PATH, CORE_NEXT_IMPORT_PATH } from "../src/consts";
+import { CORE_IMPORT_PATH, CORE_NEXT_IMPORT_PATH, CORE_TYPES_IMPORT_PATH } from "../src/consts";
 
-export type VibeCorePath = typeof CORE_IMPORT_PATH | typeof CORE_NEXT_IMPORT_PATH;
+export type VibeCorePath = typeof CORE_IMPORT_PATH | typeof CORE_NEXT_IMPORT_PATH | typeof CORE_TYPES_IMPORT_PATH;
```
Comment (YossiSaadi) [inline packages/codemod/types/paths.types.ts:3]: I think I made a mistake here
Should it be this instead?:
```suggestion
export type VibeCorePath = CORE_IMPORT_PATH | CORE_NEXT_IMPORT_PATH | CORE_TYPES_IMPORT_PATH;
```

File: packages/codemod/transformations/core/v2-to-v3/__tests__/type-imports-migration.test.ts
Comment (YossiSaadi) [inline packages/codemod/transformations/core/v2-to-v3/__tests__/type-imports-migration.test.ts:1]: same as the packages renaming, is this enough tests?
e.g. doesn't change unrelated imports, changes multiple imports, only change required when both types and core are in use, etc.

I'm saying it since we should be extra careful imo with codemods, as a wrong change/failure is a critical issue for the flow of running all the codemods

**Review by YossiSaadi — APPROVED** [review]
Looks cool, I think we can benefit from more tests with everyday cases and some edge cases
